fluent-langneg/test/locale_test.js

```diff
@@ -51,11 +51,11 @@ suite('Parses simple locales', () => {
       variant: 'mac'
     }));
 
-    assert.ok(isLocaleEqual('lij-Arab-FA-linux', {
+    assert.ok(isLocaleEqual('lij-Arab-FA-lin', {
```

Comment:
Added to https://bugzilla.mozilla.org/show_bug.cgi?id=1345957
I'll fix it in C++ and then backport here.